pages/index.tsx

```diff
@@ -14,13 +15,16 @@ interface LocaleSelectButtonProps {
   onClick: (e: React.SyntheticEvent) => void;
 }
 
-const siteTitle = 'CuentaNos';
+const siteTitle = 'Refugee.info';
 const message =
-  'Información confiable para el empoderamiento de la población en el norte de Centroamérica';
+  'A European information service for refugees, migrants, and asylum-seekers.';
 const sites: Site[] = [
-  { name: 'El Salvador', url: 'https://elsalvador.cuentanos.org/es' },
-  { name: 'Guatemala', url: 'https://guatemala.cuentanos.org/es' },
-  { name: 'Honduras', url: 'https://honduras.cuentanos.org/es' },
+  { name: 'Bulgaria', url: 'https://migrantlife.bg' },
+  { name: 'Czechia', url: 'https://czechia.refugee.info/' },
+  { name: 'Greece', url: 'https://greece.refugee.info' },
+  { name: 'Hungary', url: 'https://hungary.refugee.info' },
+  { name: 'Italy', url: 'https://italy.refugee.info' },
+  { name: 'Slovakia', url: 'https://ukraineslovakia.sk' },
```

Comment:
why are we changing the countries here? is this not going to be for latin america anymore?

Reply:
I'm not sure what happened here, all I changed was the signpost-base version, not any of this. I'm going to close and push again